### PR TITLE
Vcb 19 build system

### PR DESCRIPTION
**Pull request by ni507**

### CODE REVIEW

File: test/dummy.test.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import {add} from "../src/dummy/dummy";
```
Comment (clooudch) [inline test/dummy.test.ts:1]: I guess you could use "dummy/dummy" here, since "./src" is the base path.

File: jest.config.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
```
Comment (clooudch) [inline jest.config.js:1]: I guess this file could be written in TypeScript as well.

Comment (clooudch) [reply jest.config.js:1]: Maybe "@types/jest" needs to be installed as a devDependency.

File: test/dummy.test.ts (hunk 1 above)
Comment (ni507) [reply test/dummy.test.ts]: Changed.

File: jest.config.js (hunk 2 above)
Comment (ni507) [reply jest.config.js]: Changed.

**Review by clooudch — APPROVED** [review]

**Pull request merged**